src/bin/new-release.rs

```diff
@@ -1,13 +1,22 @@
 extern crate contributors;
 
 extern crate diesel;
+extern crate clap;
 
 use diesel::prelude::*;
-
-use std::process::Command;
-use std::env;
+use clap::{App,Arg};
 
 fn main() {
+    let matches = App::new("new-release")
+        .about("update contributors from a new release")
+        .arg(Arg::with_name("filepath")
+            .short("p")
+            .long("path")
+            .help("filepath of rust src folder")
```

Comment:
This is slightly ambiguous since Rust's source also has a `src` folder in it. Maybe

> filepath of the rust source code

?